mdbook-mermaid is missing from docs build+deploy workflow

Causing docs [publishing to fail](https://github.com/containers/bootc/actions/runs/8973312117/job/24643169427#step:5:16)

```
Run cd docs && mdbook build
2024-05-06 17:38:41 [INFO] (mdbook::book): Book building has started
2024-05-06 17:38:41 [WARN] (mdbook::preprocess::cmd): The command wasn't found, is the "mermaid" preprocessor installed?
2024-05-06 17:38:41 [WARN] (mdbook::preprocess::cmd): 	Command: mdbook-mermaid
2024-05-06 17:38:41 [INFO] (mdbook::book): Running the html backend
Error: -06 17:38:41 [ERROR] (mdbook::utils): Error: Rendering failed
Error: -06 17:38:41 [ERROR] (mdbook::utils): 	Caused By: Unable to copy across additional CSS and JS
Error: -06 17:38:41 [ERROR] (mdbook::utils): 	Caused By: Unable to copy /home/runner/work/bootc/bootc/docs/mermaid.min.js to /home/runner/work/bootc/bootc/docs/book/mermaid.min.js
Error: -06 17:38:41 [ERROR] (mdbook::utils): 	Caused By: No such file or directory (os error 2)
Error: Process completed with exit code 101.
```

The `mdbook-mermaid` requirement was added in #503 but we forgot to add it to the docs workflow.

--- a/.github/workflows/docs.yml
+++ b/.github/workflows/docs.yml
@@ -28,11 +28,18 @@ jobs:
           mkdir mdbook
           curl -sSL $url | tar -xz --directory=./mdbook
           echo `pwd`/mdbook >> $GITHUB_PATH
+      - name: Install mdbook-mermaid
+        run: |
+          tag=$(curl 'https://api.github.com/repos/badboy/mdbook-mermaid/releases/latest' | jq -r '.tag_name')
+          url="https://github.com/badboy/mdbook-mermaid/releases/download/${tag}/mdbook-mermaid-${tag}-x86_64-unknown-linux-gnu.tar.gz"
+          mkdir mdbook-mermaid
+          curl -sSL $url | tar -xz --directory=./mdbook-mermaid
+          echo `pwd`/mdbook-mermaid >> $GITHUB_PATH
       - name: Setup Pages
         id: pages
         uses: actions/configure-pages@v4
       - name: Build with mdBook
-        run: cd docs && mdbook build
+        run: cd docs && mdbook-mermaid install && mdbook build
       - name: Upload artifact
         uses: actions/upload-pages-artifact@v3
         with:

